lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -37,6 +37,19 @@ using namespace mlir::torch;
 
 namespace {
 
+Type toTorchTensorTypeCast(PatternRewriter &rewriter, Type ty) {
+  auto tensorTy = ty.cast<TensorType>();
+
+  auto elementType = dyn_cast<IntegerType>(tensorTy.getElementType());
+  if (elementType && elementType.isSignlessInteger()) {
+    elementType = rewriter.getIntegerType(tensorTy.getElementType().getIntOrFloatBitWidth(), true);
```

Comment:
```suggestion
    elementType = rewriter.getIntegerType(tensorTy.getElementType().getIntOrFloatBitWidth(), /*isSigned=*/true);
```
(guess this will need reformatting)

Reply:
This file was throwing my clangd off the tracks.. I can check formatting as a last action item (to avoid making the diff noisy)

Reply:
Just to clarify: We usually don't re-format the whole file, but the newly added changes should be properly formatted.

Reply:
Done